media/css/test.scss

```diff
@@ -0,0 +1,3 @@
+#nonexistent ul > li {
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 3